setup.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
```

Comment:
This was purposefully removed with an explanation here: https://github.com/ansible/ansible/pull/18171#issuecomment-255861934

I can't remember which pypa document had the advice to not contain a shebang in setup.py but the pypa example setup.py does indeed not contain the shebang: https://github.com/pypa/sampleproject/blob/master/setup.py

Reply:
It's harmless, but I don't mind excluding it from this patch